### PR TITLE
Adds documentation for media-upload.js

### LINKED ISSUE #11: Document wp-admin/js/media-upload.js

### PR DESCRIPTION
**Pull request by diedexx**
Fixes #11 
Worked on this together with @jcomack.

### CODE REVIEW

File: src/wp-admin/js/media-upload.js [hunk 1]
```diff
@@ -1,13 +1,36 @@
...
 
+/**
+ * @summary Sends the HTML passed in the parameters to TinyMCE.
+ *
+ * @since 2.3.3
+ *
+ * @global {tinymce} tinymce An instance of the TinyMCE editor.
+ * @global {QTags} QTags An instance of QTags.
+ *
+ * @param {string} html The HTML/text to be sent to the editor.
```
Comment (terw-dan) [inline src/wp-admin/js/media-upload.js]: Can we assume it is either HTML or text that is sent? Isn't this always treated as HTML from and to the editor?

**Comment by andizer** [comment]
@diedexx @jcomack  What is the status of this pr? There is no label attached...

**Comment by diedexx** [comment]
I believe it needs another code reivew @andizer .

File: src/wp-admin/js/media-upload.js [hunk 2]
```diff
@@ -55,6 +88,11 @@ var tb_position;
 				tbWindow.css({'top': 20 + adminbar_height + 'px', 'margin-top': '0'});
 		}
 
+		/**
+		 * @summary Removes any width and height parameters from the href of any anchor with the ThickBox class and recalculates the new height and width values.
+		 *
+		 * @since 2.6.0
+		 */
 		return $('a.thickbox').each( function() {
```
Comment (terw-dan) [inline src/wp-admin/js/media-upload.js]: Please move this to the correct function. Probably the code was updated so the documentation block is now in the wrong position.

File: src/wp-admin/js/media-upload.js [hunk 3]
```diff
@@ -55,6 +88,11 @@ var tb_position;
 				tbWindow.css({'top': 20 + adminbar_height + 'px', 'margin-top': '0'});
 		}
 
+		/**
+		 * @summary Removes any width and height parameters from the href of any anchor with the ThickBox class and recalculates the new height and width values.
```
Comment (terw-dan) [inline src/wp-admin/js/media-upload.js]: Please make sure each line doesn't exceed the 80 character limit.

**Comment by atimmer** [comment]
Committed in https://core.trac.wordpress.org/changeset/42426.

**Pull request closed without merging**